Gulpfile.ts

```diff
@@ -551,7 +551,7 @@ function restoreSavedNodeEnv() {
     process.env.NODE_ENV = savedNodeEnv;
 }
 
-let testTimeout = 20000;
+let testTimeout = 40000;

```

Comment:
opps I am using mac and the machine time out. I can roll back though would love to keep it so I can run on my mac :smile_cat: